run.py

```diff
@@ -932,12 +932,23 @@ def collect_recipe(ceph_cluster):
     """
     version_datails = {}
     client_node = ceph_cluster.get_ceph_objects("client")
+
     out, rc = client_node[0].exec_command(
         sudo=True, cmd="podman --version | awk {'print $3'}"
     )
     output = out.read().decode().rstrip()
-    log.info(f"Podman Version {output}")
-    version_datails["PODMAN"] = output
+    if output:
+        log.info(f"Podman Version {output}")
+        version_datails["PODMAN"] = output
+
+    out, rc = client_node[0].exec_command(
+        sudo=True, cmd="docker --version | awk {'print $3'}"
+    )
+    output = out.read().decode().rstrip()
+    if output:
+        log.info(f"Docker Version {output}")
+        version_datails["DOCKER"] = output
+
     out, rc = client_node[0].exec_command(
         sudo=True, cmd="ceph --version | awk {'print $3'}"
```

Comment:
In case of 5.x, you would need to execute this command in `cephadm` if we plan to move to using the installer node.

Reply:
Hi Satya,
In both 4.x and 5.0 we are not installing ceph-common on the installer node.
So for getting ceph version I am using the client node.
for the rest of the things I have used the Installer node